demo/config/config.exs

```diff
@@ -0,0 +1,98 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+
+import Config
+
+config :demo,
+  ecto_repos: [Demo.Repo],
+  generators: [timestamp_type: :utc_datetime]
+
+config :demo, DemoWeb.Endpoint,
```

Comment:
Will merge with the below endpoint definition

Or should I introduce `dev.exs`? I was thinking, that the demo app will only ever be in use in development, so I wanted to keep the config simple